components/rss/rss.app.mjs

```diff
@@ -0,0 +1,67 @@
+import { axios } from "@pipedream/platform";
+import FeedParser from "feedparser";
+import hash from "object-hash";
+import stringToStream from "string-to-stream";
```

Comment:
Please update the `package.json` with the modules.
You can use `npm install --save yourmodule` on the root of the `rss`

Reply:
done